core/src/test/java/org/apache/calcite/test/RelOptRulesTest.java

```diff
@@ -6925,6 +6925,13 @@ private HepProgram getTransitiveProgram() {
         .check();
   }
 
+  @Test void testProjectAggregateMergeSum01() {
```

Comment:
Add a hyperlink to jira case.  It will help explain what problem the test case is solving.